packages/core/src/ScaledText/ScaledText.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export type IScaledTextProps = {
+  id?: React.Key
+  lines: {
+    text: string
+    key?: React.Key
+    props?: React.SVGProps<SVGTextElement>
+  }[]
+  color?: string
+}
+
+const ScaledText = ({ id, lines, color }: IScaledTextProps) => {
+  const _id = id ?? uuidv4()
```

Comment:
The ID should probably be set on state, otherwise, it seems like this could cause unnecessary rerenders.

Reply:
Thanks for the 👀! Added id to State to prevent any unnecessary rerenders!